src/reference/00-Getting-Started/07A-Scopes.md

```diff
@@ -379,104 +379,153 @@ lazy val util = (project in file("util"))
+##### Rule 1: Scope precedence
 
-- Rule 1A: If a scope has more specific Subproject axis, then it will always have higher precedence over other scopes without.
-- Rule 1B: If a scope has more specific Dependency configuration, then it will have higher precedence over specific task scoping.
+The first rule of scope delegation is that scope axes have the following _precedence_:
+subproject axis, dependency configuration axis, and task axis.
 
-What is the expected `name in projB` value in the following build?
+Let's look at an example. Given the following build definition:
 
 ```scala
-scalaVersion in (ThisBuild, packageBin) := "2.12.2"
+val bippy = settingKey[String]("The biddy identifier")
+
+bippy in Global := "abc"
 
 lazy val projB = (project in file("b"))
   .settings(
-    name := {
-      "foo" + (scalaVersion in packageBin).value
-    },
-    scalaVersion := "2.11.11"
+    bippy := "xyz",
+    name := "foo-" + (bippy in packageBin).value
   )
 ```
 
-The answer is `foo2.11.11`. Because of Rule 1A `scalaVersion` scoped to `projB` has
-higher precendence over `(ThisBuild, packageBin)`.
+What is the value of `name in projB`?
+
+1. `foo-abc`?
+2. `foo-xyz`?
+3. something else?
+
+The answer is `foo-xyz`, because `bippy` scoped to `projB` has
+higher precendence to the `bippy in Global` setting.
 
-Next, what would you see if you ran `projC/test`?
+Let's look at another example. Given the following build definition:
 
 ```scala
 scalacOptions in ThisBuild += "-Ywarn-unused-import"
 
 lazy val projC = (project in file("c"))
   .settings(
-    test := {
-      println((scalacOptions in (Compile, console)).value)
-    },
+    scalacOptions in Compile := scalacOptions.value, // sbt defines and adds this by default
     scalacOptions in console -= "-Ywarn-unused-import",
-    scalacOptions in Compile := scalacOptions.value // added by sbt
+    test := println((scalacOptions in (Compile, console)).value)
   )
 ```
 
-The answer is `List(-Ywarn-unused-import)`. Because of Rule 1B `scalacOptions in Compile`
-has higher precedence than `scalacOptions in console`.
+What would you see if you ran `projC/test`?
 
-- Rule 2: For Dependency configuration axis, sbt searches in the order of the given value,
-  its parents, their parents recursively, and then falls back to `Global`.
+1. `List()`?
+2. `List(-Ywarn-unused-import)`?
+3. something else?
 
-The example for that is `projA` that we saw earlier:
+The answer is `List(-Ywarn-unused-import)`. The reason is sbt will only use the `scalacOptions in Compile`
+setting, and *never* the `scalacOptions in console` setting, because settings with defined dependency
+configuration scopes have higher precendence to settings defined in `Global` dependency configuration scope.
+
+##### Rule 2: Subproject scope axis delegation
+
+- Rule 2: For the subproject axis, sbt will delegate settings scoped in the target subproject, then to
+    `ThisBuild`, and finally fall back to `Global`.
+
+Here's an example:
+
+```scala
```

Comment:
I'm generally confused about this example. I don't think it even compiles because of the use of infix.